tests/components/zwave_js/conftest.py

```diff
@@ -0,0 +1,59 @@
+"""Provide common Z-Wave JS fixtures."""
+import json
+from unittest.mock import patch
+
+import pytest
+from zwave_js_server.model.driver import Driver
+from zwave_js_server.model.node import Node
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+@pytest.fixture(name="basic_data", scope="session")
+def basic_data_fixture():
+    """Load the basic fixture data."""
+    return load_fixture("zwave_js/basic_dump.txt").split("\n")
+
+
+@pytest.fixture(name="client")
+def mock_client_fixture(basic_data):
+    """Mock a client."""
+    state = json.loads(basic_data[0])["state"]
```

Comment:
You should put this in a fixture. Now if you hav a mock_client and a multisensor fixture, it's loaded twice. 

You want this fixture to be loaded once per session maybe ?

Reply:
Oh wait, you did do that with basic data. Just not the JSON parsing.

Reply:
Yeah, I need to select the data to load.